RESOURCES/INTHEWILD.md

```diff
@@ -172,6 +172,7 @@ Join our growing community!
 
 ### Travel
 - [Skyscanner](https://www.skyscanner.net/) [@cleslie, @stanhoucke]
+- [Agoda](https://www.agoda.com/) [@maiake, @obombayo]
```

Comment:
Please reorder in alphabetical order (A before S)

Reply:
@villebro Done.